lib/Settings/Admin.php

```diff
@@ -34,7 +34,7 @@ public function getForm() {
 		foreach ($providerIds as $id => $name) {
 			$providers[] = [
 				'id' => $id,
-				'name' => $name === '' ? $this->l10n->t('Provider ') . $id : $name
+				'name' => $name === '' ? $this->l10n->t('Provider %s', $id)  : $name
```

Comment:
```suggestion
				'name' => $name === '' ? $this->l10n->t('Provider %s', [$id])  : $name
```

Oh, one more, second parameter must be a string or an array, given the psalm error reported. Preferably array I'd say.